lib/rdoc/ri/driver.rb

```diff
@@ -1152,13 +1152,36 @@ def interactive
   # Is +file+ in ENV['PATH']?
 
   def in_path? file
-    return true if file =~ %r%\A/% and File.exist? file
+    extensions = [''].union(RbConfig::CONFIG["EXECUTABLE_EXTS"]&.split || RbConfig::CONFIG["EXEEXT"].to_a)
```

Comment:
`RbConfig::CONFIG["EXECUTABLE_EXTS"]` is set since 1.9.2.
`RbConfig::CONFIG["EXEEXT"]` is the single suffix to be built, not a list.

Reply:
`RbConfig::CONFIG["EXECUTABLE_EXTS]` looks good enough and stable, I put `RbConfig::CONFIG["EXEEXT"]` there just in case.
But I tried ruby implementations other than MRI, JRuby does not have `RbConfig::CONFIG["EXECUTABLE_EXTS]`.
And I found `String` does not have `to_a`, it translate `nil` into `[]` if `RbConfig::CONFIG` not have `"EXEEXT"`. But it's broken.
`[RbConfig::CONFIG["EXEEXT"]]` insted of `RbConfig::CONFIG["EXEEXT"].to_a` fix this.

As another point of view, `RDoc::RI::Driver#find_pager_jruby`, that is used to create pipe for JRuby, seems to execute childprocess that has filename extension `.exe` `.com` `.bat` `.cmd`. But it can omit filename extension only `.exe`. Even if in_path? compensate `.com`, JRuby not yet found `more.com` when `set PAGER=more`. `set PAGER=more.com` is required.
I'm not familiar to Java and JRuby, I don't know which mechanism is doing this.

I think changing commandline filename in PAGER introduces complexity and I don't want to fall into the quote nightmare.
For JRuby on Windows, it's a reasonable choice to set PAGER to the filename with extension other than `.exe`. 
And add `more.com` fallback, it supports for most use cases.